catalog/pages/inputs/index.md

```diff
@@ -138,30 +138,30 @@ span: 6
     <Row>
         <Column medium={6} style={{ padding: "16px 0" }}>
           <RadioGroup aria-label="test large radio">
-                <RadioButton size="large" name="Option1" value="FirstButton" index={0} isTopAligned>
+                <RadioButton size="large" name="example1" value="FirstButton" index={0} isTopAligned>
```

Comment:
Use **Option1** instead of **example1**.

Reply:
Since it's a group of different options depicting one particular example so **example1** sounds better to me.

Reply:
Okay.